cloud/docker/docker_compose

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/python
+
+# (c) 2015, Sam Yaple <sam@yaple.net>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: docker_compose
+version_added: 1.8.4
+short_description: Manage docker containers with docker-compose
+description:
+     - Manage the lifecycle of groups of docker containers with docker-compose
+options:
+  command:
+    description:
+      - Select the compose action to perform
+    required: true
+    choices: ['build', 'kill', 'pull', 'rm', 'scale', 'start', 'stop', 'restart', 'up']
+  compose_file:
+    description:
+      - Specify the compose file to build from
+    required: true
+    type: bool
+  insecure_registry:
+    description:
+      - Allow access to insecure registry (HTTP or TLS with a CA not known by the Docker daemon)
+    type: bool
+  kill_signal:
+    description:
+      - The SIG to send to the docker container process when killing it
+    default: SIGKILL
+  no_build:
+    description:
+      - Do not build an image, even if it's missing
+    type: bool
+  no_deps:
+    description:
+      - Don't start linked services
+    type: bool
+  no_recreate:
```

Comment:
By default, docker-compose recreates containers when rerun (stops/destroys/starts). This isn't really the desired behavior for ansible modules as I understand since it actually forces a change each time.

Thoughts?